packages/frontend/src/components/LightdashVisualization/index.tsx

```diff
@@ -8,19 +8,16 @@ import { useVisualizationContext } from './VisualizationProvider';
 const LightdashVisualization: FC = () => {
     const { chartType } = useVisualizationContext();
 
-    const renderType = () => {
-        switch (chartType) {
-            case ChartType.BIG_NUMBER:
-                return <SimpleStatistic />;
-            case ChartType.TABLE:
-                return <SimpleTable />;
-            case ChartType.CARTESIAN:
-                return <SimpleChart />;
-            default:
-                return null;
-        }
-    };
-    return <>{renderType()}</>;
```

Comment:
just curious, why did you change this ?

Reply:
Just removing redundant code and because calling a function inside the jsx is a bad practice.